metasyn/var.py

```diff
@@ -47,82 +48,23 @@ class MetaVar():
         User-provided description of the variable.
     """
 
-    dtype = "unknown"
-
     def __init__(self,  # pylint: disable=too-many-arguments
+                 name: str,
                  var_type: str,
-                 series: Optional[Union[pl.Series, pd.Series]] = None,
-                 name: Optional[str] = None,
-                 distribution: Optional[BaseDistribution] = None,
-                 prop_missing: Optional[float] = None,
-                 dtype: Optional[str] = None,
-                 description: Optional[str] = None):
+                 distribution: BaseDistribution,
+                 dtype: str = "unknown",
+                 description: Optional[str] = None,
+                 prop_missing: float = 1.0):
```

Comment:
default 1.0?? is this for "data-free"? otherwise I'd rather expect prop_missing to be 0.0 by default.